src/osd/osd_types.h

```diff
@@ -3677,6 +3677,30 @@ struct pg_log_t {
     return divergent;
   }
 
+  void merge_from(const vector<pg_log_t*>& slogs, eversion_t last_update) {
+    log.clear();
```

Comment:
I think we want to do something similar to what we do during trimming - convert the last n log entries to dups, especially since our default steady state will have only log entries, no dups

Reply:
since we changed the default max log size and max dups to be equal